python/cudf/cudf/tests/test_numerical.py

```diff
@@ -194,6 +194,7 @@ def test_to_numeric_downcast_int(data, downcast):
     assert_eq(expected, got)
 
 
+@pytest.mark.filterwarnings("ignore:invalid value encountered in cast")
```

Comment:
Instead of applying this to the whole test, can we just wrap the `pd.to_numeric` call? This doesn't affect the `cudf.to_numeric` call, does it?

Also, should we be handling the warning conditionally? i.e. I assuming this happens when trying to downcast a signed to an unsigned type or something?